src/dashboard/router/modules/products.js

```diff
@@ -60,11 +60,14 @@ class ProductsResource extends Resources {
     }
 
     if (sort) {
-      // replace sort rule
       if (body.sort.length > 4) {
         body.sort[2] = sort
       } else {
-        body.sort.splice(2, 0, sort)
+        if ('quantity' in sort || 'price' in sort) {
```

Comment:
```suggestion
        if (sort.quantity || sort.price) {
```

Evita erros caso o `sort` não seja um objeto, não sei se está implementado desta forma mas o ELS também aceita strings no em `dsl.sort`